package.json

```diff
@@ -1,7 +1,7 @@
 {
   "dependencies": {
     "babel-cli": "^6.16.0",
-    "babel-preset-steelbrain": "^5.0.1",
+    "@mcro/preset-motion": "5.0.2",
```

Comment:
If you create a new app with motion and then change the babelrc to use @mcro/preset-motion, you'll see it won't boot up due to error where it's looking for `babel-preset-steelbrain`. I imagine this is the case for all other presets and is a bug somewhere. The solution is obviously not this, I was just pushing my local fix :/